website/docs/getting-started/for-engineers.md

```diff
@@ -262,6 +262,55 @@ Our styles, components and icons are supported by the following browsers:
 | Firefox        | last 2 versions |
 | Microsoft Edge | last 2 versions |
 
+## Internationalization
+
+We use [`ember-intl`](https://github.com/ember-intl/ember-intl) to handle internationalization in HDS components.
+
+### Default behavior
+
+By default, HDS components display text in English when `ember-intl` is not installed in your application. This ensures components work out of the box without additional configuration.
+
+### Setting up internationalization
+
+To enable internationalization in your application:
+1. Install `ember-intl` as a dependency in your application:
+```bash
+pnpm add ember-intl
+```
+2. Configure your application to use one of the supported HDS translation locales.
+For detailed setup instructions, refer to the [ember-intl quickstart guide](https://ember-intl.github.io/ember-intl/docs/quickstart).
+
+### Translation support
+
+#### Using HDS-provided translations
+
+You can use translations provided by HDS by setting your application's locale to a supported value. Components will automatically display the appropriate translation for that locale.
+
+Currently supported locales in HDS:
+
+- `en-us` (English - United States)
```

Comment:
This is cool. Do we have any current plans/prioritization around adding additional specific languages?